R/test_wrapper.R

```diff
@@ -1347,20 +1347,32 @@ exp_anova <- function(df, var1, var2, covariates = NULL, func2 = NULL, covariate
     power <- 1.0 - beta
   }
   var1_col <- col_name(substitute(var1))
-  var2_col <- col_name(substitute(var2))
+  var2_ <- substitute(var2)
```

Comment:
Add a comment for var1 and var2 at the function declaration.

Reply:
Done.